Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate_test.py

```diff
@@ -1,14 +1,20 @@
 import pytest
 import requests
-from requests import RequestException
 
 import demistomock as demisto
 import importlib
-from CommonServerPython import DBotScoreReliability
+from CommonServerPython import DBotScoreReliability, ExecutionMetrics
 
 Cisco_umbrella_investigate = importlib.import_module('Cisco-umbrella-investigate')
 
 
+def get_mterics_test(expected_len_of_results):
+    is_supported = ExecutionMetrics.is_supported()
```

Comment:
please mock the server versions for cases of below and above the supported version (unittests don't run on an actual XSOAR server).